src/qt_gui/main_window_ui.h

```diff
@@ -0,0 +1,279 @@
+// SPDX-FileCopyrightText: Copyright 2024 shadPS4 Emulator Project
```

Comment:
Is there a reason to include the _ui file manually? Qt generates those automatically.

Reply:
this is converted from *.ui file to cpp using qt tool , the purpose is to eliminate the use of *.ui files . this will be rewritten better though